sentry/src/main/java/io/sentry/TransactionContext.java

```diff
@@ -30,11 +30,12 @@ public static TransactionContext fromPropagationContext(
       baggage.freeze();
 
       Double sampleRate = baggage.getSampleRateDouble();
-      Boolean sampled = parentSampled != null ? parentSampled.booleanValue() : false;
-      if (sampleRate != null) {
-        samplingDecision = new TracesSamplingDecision(sampled, sampleRate);
-      } else {
-        samplingDecision = new TracesSamplingDecision(sampled);
+      if (parentSampled != null) {
```

Comment:
now uses `null` sampling decision instead of `false` for incoming sentry-trace header with deferred sampling decision. This causes sentry sampler to make a decision instead of assuming `false` from parent should be copied.